en/admin/config.rst

```diff
@@ -2209,6 +2213,10 @@ The following are other parameters. The type and value range for each parameter
 
     **supplemental_log** is a parameter to determine whether information needed to support the CDC (Change Data Capture) or :ref:`flashback` is written to the log volume. CDC and flashback must be able to see how transactions logically changed the database through the physical logs. Any additional information required to interpret the physical logs is saved as **supplemental_log** . Setting this parameter bigger than 0 affects performance and log space because more logs are created and stored in addition to the existing transaction logs. If this parameter is set to 1, the information necessary to interpret DML and DDL executed by the user is logged. If it is set to 2, only information necessary to interpret the DML is logged.
 
+**regexp_engine**
```

Comment:
I think it should be mentioned that the types of values that can be set are "re2" and "cppstd".
No mention of "cppstd".

Reply:
The following paragraph is written. Wouldn't that be enough?
`**cppstd** or **re2** can be set and the default value is **re2**. `

Reply:
I didn't see the paragraph below. Enough.